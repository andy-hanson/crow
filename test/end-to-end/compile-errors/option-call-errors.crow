main void()
	int int = -1
	int-opt int? = -1,
	x nat? = int?.abs
	y nat = int-opt?.abs
	z = int?.abs
	()
