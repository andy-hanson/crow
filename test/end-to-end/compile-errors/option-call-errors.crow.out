exit code: 1
stderr:
test/end-to-end/compile-errors/option-call-errors.crow 4:14-4:17 Expected type 'int64?'.
Actual: 'int64'.
test/end-to-end/compile-errors/option-call-errors.crow 5:13-5:25 Expected type 'nat64'.
Actual: 'nat64?'.
test/end-to-end/compile-errors/option-call-errors.crow 6:9-6:12 Expected one of these types:
	'int64?'
	'float32?'
	'float64?'
Actual: 'int64'.
Uncaught exception: Reached compile error
	at main
